packages/docs/.vitepress/components/Banner.vue

```diff
@@ -2,57 +2,78 @@
   <a
     v-if="isVisible"
     id="vs"
-    href="https://vuejsforge.com/?friend=vuerouter&utm_source=pinia&utm_medium=website&utm_campaign=affiliate&utm_content=top_banner"
+    href="https://vueschool.com/sales/vuejsforge?friend=vuerouter&utm_source=vuerouter&utm_medium=website&utm_campaign=affiliate&utm_content=top_banner"
```

Comment:
@nicodevs @posva is the utm_source=vue-router correct?

Reply:
@JeraldVin  it is, that's the referral ID that Pinia and Vue Router share :slightly_smiling_face: